source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -113,6 +113,13 @@ destination cluster.
 Chunk Distribution
 ''''''''''''''''''
 
+.. important::
+   
+   Even if the source cluster is balanced, ``mongosync`` does not
+   ensure balance of the destination cluster. To balance the destination
```

Comment:
Since Sharding operations are not permitted during the execution of mongosync (see [Limitations](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/limitations/)), you must wait until it is safe to accept writes in order to rebalance the destination cluster. See [Sharded Cluster Balancer](https://www.mongodb.com/docs/manual/core/sharding-balancer-administration/#sharded-cluster-balancer) for guidance on how to rebalance the cluster.